subliminal/providers/legendastv.py

```diff
@@ -164,6 +164,13 @@ class LegendasTVProvider(Provider):
     subtitle_class = LegendasTVSubtitle
 
     def __init__(self, username=None, password=None):
+
+        # Provider needs UNRAR installed. If not available raise ConfigurationError
+        try:
+            rarfile._check_unrar_tool()
```

Comment:
I didn't say they weren't valid, I said I don't think it raises that exception,
because look at the code in the first link.
```python
try:
    # does UNRAR_TOOL work?
    custom_check([ORIG_UNRAR_TOOL], True)
except RarCannotExec:
    try:
        # does ALT_TOOL work?
        custom_check([ALT_TOOL] + list(ALT_CHECK_ARGS), True)
    except RarCannotExec:
        # no usable tool, only uncompressed archives work
        pass
```

It passes, meaning - doesn't raise it.
It will only raise RarExecError which is the base exception for RarCannotExec.
I'm not sure how Python handles that.